### PR TITLE
Mission-list: Milestone 1

### PR DESCRIPTION
**Pull request by Gathin23**

### CODE REVIEW

File: src/components/ServiceItem.tsx [hunk 1]
```diff
@@ -9,61 +9,102 @@ function limitText(text: string, maxLength: number) {
...
 
-function ServiceItem({ service, embedded }: { service: IService; embedded?: boolean }) {
+function ServiceItem({
+  service,
+  embedded,
+  view,
+}: {
+  service: IService;
+  embedded?: boolean;
+  view?: number;
```
Comment (0xRomain) [inline src/components/ServiceItem.tsx:19]: best practices: need an Enum for typing

File: src/components/ServiceList.tsx [hunk 2]
```diff
@@ -35,18 +36,71 @@ function ServiceList() {
...
 
-      <div className='flex justify-center items-center gap-10 flex-col pb-5'>
-        <SearchServiceButton value={searchQuery} />
-      </div>
+      <div className='flex flex-col md:flex-row mb-5'>
+        <div className='flex mb-2 md:mb-0'>
+          <button
+            onClick={() => setView(1)}
+            className={`px-4 py-2 rounded-full ${
+              view === 1 ? 'bg-primary text-white' : 'bg-transparent text-base-content'
```
Comment (0xRomain) [inline src/components/ServiceList.tsx]: you can't use classes that are not part of the user palette. Here it mus best text-primary.
For example here if user setup primary as white, the button is white on white..

You can't use also bg-transparent

Comment (0xRomain) [reply src/components/ServiceList.tsx]: check this on all your integration, i saw this several time

File: src/components/ServiceList.tsx [hunk 3]
```diff
@@ -35,18 +36,71 @@ function ServiceList() {
...
+          <ServiceItem
+            service={service}
+            embedded={router.asPath.includes('embed/')}
+            key={i}
+            view={view}
+          />
+        ))}
+
+      {view === 2 && (
+        <table className='min-w-full text-center' style={{ borderRadius: '1rem' }}>
```
Comment (0xRomain) [inline src/components/ServiceList.tsx]: there are classesfor borderRadius

File: src/components/ServiceItem.tsx (hunk 1 above)
Comment (Gathin23) [reply src/components/ServiceItem.tsx:19]: Yes but the previous code and also the rest of the components follow similar format

**Pull request merged**